_pages/rules.html

```diff
@@ -31,9 +31,13 @@ <h2 class="page-heading">B. General Gameplay</h2>
         <ol>
           <li>Players may not abuse glitches, like placing blocks in areas restricted by the plugin to walk on, also known as "block glitching".</li>
           <li>Maps may have rules that apply only to that specific map. They can be read under <code>/map</code>.</li>
-          <li>Players may only press one button, and have it register as one click.</li>
-          <span class="badge badge-warning">Note</span>
-          <small>Clicking techniques, such as butterfly/jitter/drag-clicking, are discouraged as they can trigger our anti-cheat in such a manner than indicates auto-clicking. Do so at your own risk.</small>
+          <li>Players may only press one button, and have it register as one click. Any behaviour that results in double clicks is forbidden.</li>
+          <ol>
```

Comment:
```suggestion
          <ol type="a">
```
This would make the nested list use a, b, c etc instead of another numbering.